packages/datadog_flutter_plugin/lib/src/logs/ddlogs_web.dart

```diff
@@ -0,0 +1,159 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+// ignore_for_file: unused_element
+
+@JS('DD_LOGS')
+library ddlogs_flutter_web;
+
+import 'package:js/js.dart';
+
+import '../../datadog_flutter_plugin.dart';
+import '../web_helpers.dart';
+import 'ddlogs_platform_interface.dart';
+
+class DdLogsWeb extends DdLogsPlatform {
+  final Map<String, Logger> _activeLoggers = {};
+
+  static void initLogs(DdSdkConfiguration configuration) {
+    String? version = configuration.additionalConfig[DatadogConfigKey.version];
+
+    init(_LogInitOptions(
+      clientToken: configuration.clientToken,
+      env: configuration.env,
+      site: siteStringForSite(configuration.site),
+      proxyUrl: configuration.customEndpoint,
+      service: configuration.serviceName,
+      version: version,
+    ));
+  }
+
+  @override
+  Future<void> createLogger(
+      String loggerHandle, LoggingConfiguration config) async {
+    var loggerHandlers = [
+      if (config.sendLogsToDatadog) 'http',
+      if (config.printLogsToConsole) 'console'
+    ];
+    var logger = _createLogger(
+      config.loggerName ?? 'default',
+      _JsLoggerConfiguration(),
+    );
+    if (logger != null) {
+      if (loggerHandlers.isNotEmpty) {
+        logger.setHandler(loggerHandlers);
+      } else {
+        logger.setHandler(['silent']);
+      }
+
+      _activeLoggers[loggerHandle] = logger;
+    }
+  }
+
+  @override
+  Future<void> addAttribute(
+      String loggerHandle, String key, Object value) async {
+    final logger = _activeLoggers[loggerHandle];
+    logger?.addContext(key, valueToJs(value, 'value'));
+  }
+
+  @override
+  Future<void> addTag(String loggerHandle, String tag, [String? value]) async {}
```

Comment:
nit: maybe we can print _"not supported"_ warning or sth else to indicate users that it doesn't do anything.

Reply:
I thought about that, but I worry that would get really spammy adding 'not supported' for things that you use on mobile that aren't translated yet for web...